app/src/pages/inside/dashboardPage/widgets/uniqueBugsTable/foundIn/foundIn.jsx

```diff
@@ -0,0 +1,52 @@
+import { PureComponent, Fragment } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames/bind';
+import Link from 'redux-first-router-link';
+import PropTypes from 'prop-types';
+import { activeProjectSelector } from 'controllers/user';
+import TagIcon from 'common/img/tag-inline.svg';
+import Parser from 'html-react-parser';
+import styles from './foundIn.scss';
+
+export const cx = classNames.bind(styles);
+
+@connect((state) => ({
+  projectId: activeProjectSelector(state),
+}))
+export class FoundIn extends PureComponent {
+  static propTypes = {
+    value: PropTypes.object.isRequired,
+    className: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const { value, className, projectId } = this.props;
+    return (
+      <div className={cx('found-in-col', className)}>
+        {value.foundIn &&
+          value.foundIn.map((item) => {
+            const path = [item.launchId].concat(Object.keys(item.pathNames)).join('/');
+            return (
+              <Fragment key={item.id}>
+                <div className={cx('item')} title={item.name}>
+                  <Link
+                    to={`#${projectId}/launches/all/${path}?log.item=${item.id}`}
```

Comment:
We encapsulare all paths in the router config, so please use objects for links.
```
{
  type: PROJECT_LOG_PAGE,
  payload: {
    projectId,
    filterId: ALL, // from app/src/common/constants/reservedFilterIds.js
    testItemIds: path, // const path = [item.launchId].concat(Object.keys(item.pathNames), item.id).join('/');
  }
}
```

Reply:
Fixed